clientdb/sidecar.go

```diff
@@ -137,7 +137,17 @@ func (db *DB) UpdateSidecar(ticket *sidecar.Ticket) error {
 			return ErrNoSidecar
 		}
 
-		// TODO(roasbeef): remove the bid if in the final state now/
+		// If the ticket is in a terminal state, we won't ever need the
```

Comment:
Nice, optionally we _could_ attempt to go back through and "re update" existing tickets if we want to be able reclaim that space (likely a negligible amount).

Reply:
Yeah, but I don't think that many people have a lot of bids on their mainnet nodes at this point, not sure if it's worth the extra code.